CoreUtilities/src/au/gov/asd/tac/constellation/utilities/javafx/JavaFxUtilities.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2010-2025 Australian Signals Directorate
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.utilities.javafx;
+
+import javafx.scene.paint.Color;
+
+/**
+ * Utility class related to JavaFx
+ *
+ * @author Quasar985
+ */
+public class JavaFxUtilities {
```

Comment:
see my comment on need for the awt color class in the Histogram display. I suspect if you convert to only using fx colors there than this class is no longer required